packages/notifi-react-card/lib/components/intercom/ChatMessageDate.tsx

```diff
@@ -26,7 +33,9 @@ export const ChatMessageDate: React.FC<ChatMessageDateProps> = ({
           classNames?.content,
         )}
       >
-        {date}
+        {isStartDate
+          ? formatConversationStartTimestamp(createdDate)
+          : formatConversationDateTimestamp(createdDate)}
```

Comment:
Can you please check on this? It looks like you are showing the formatted date, no matter what?

Reply:
Do you mean we need to show 'today' when it's today? The logic has been included in the `formatConversationStartTimestamp`